core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildStep.java

```diff
@@ -225,6 +228,9 @@ public NativeImageBuildItem build(NativeConfig nativeConfig, NativeImageSourceJa
             } else {
                 command.add("-H:-AddAllCharsets");
             }
+            if (enableAllTimeZones) {
+                command.add("-H:+IncludeAllTimeZones");
```

Comment:
If we have a proper property then I suppose we can ad the `-` version too as the user will be able to override it.

Reply:
Sure that's the plan, as soon as we have a better way to expose this configuration.